school-calendar/module2023.json

```
@@ -0,0 +1,50 @@
+[
```

[相談]

[学年暦ページ](https://www.tsukuba.ac.jp/campuslife/calendar-school/2023.html)には

> 春季休業 筑波キャンパス 4月1日（土） ～ 4月4日（火）

とありますが、以下のように反映したほうがいいでしょうか？

```suggestion
[
  {
      "year": 2023,
      "module": "SpringVacation",
      "start": "2023-04-01",
      "end": "2023-01-04"
  },
```

---

dbスキーマの問題で2023年度にSpringVacationが2つ入れられないので入れてないです、、、
（設定しなくても実害は無い）
できたらどこかのタイミングで直した方が良いかも

---

了解です！
正直なくても実害はないのでこのままでいいと思いました。